<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RPiKeyerTerm</class>
 <widget class="QMainWindow" name="RPiKeyerTerm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RPiKeyerTerm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="receiveTextArea"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="receiveTextClearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0" rowspan="2">
         <widget class="QPlainTextEdit" name="sendTextArea">
          <property name="placeholderText">
           <string>Type the text to send and click SEND or Ctrl + Enter</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="sendButton">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>SEND</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+Return</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="sendClearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Add your copied text to the recieved text to show the conversation.  Ctrl + U&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Update</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+U</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="updateClearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QPlainTextEdit" name="conversationTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>125</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Type the text you recieve here and click Update Conv to add it to the trace above.</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="contactGroupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item row="0" column="10">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>3</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="delCallButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>RST Out</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>RST In</string>
         </property>
        </widget>
       </item>
       <item row="0" column="9">
        <widget class="QLineEdit" name="destGridLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Other station's MH grid</string>
         </property>
         <property name="maxLength">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QDoubleSpinBox" name="pwrDoubleSpinBox">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="minimum">
          <double>0.100000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
         <property name="value">
          <double>5.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="rstInLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Receiveed signal report</string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLineEdit" name="opNameLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Other operator name {OP}</string>
         </property>
         <property name="placeholderText">
          <string>Op Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QLineEdit" name="mycallLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>85</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This station's call sign {CS}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>N0CALL</string>
         </property>
         <property name="maxLength">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item row="1" column="6" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>My Call</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>PWR Out</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Loc</string>
         </property>
        </widget>
       </item>
       <item row="1" column="9">
        <widget class="QSpinBox" name="sendingSpeedSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="mhGridLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Maidenhead (or other) grid reference {MH}</string>
         </property>
         <property name="maxLength">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLineEdit" name="rstOutLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>65</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>65</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Sent signal report</string>
         </property>
         <property name="maxLength">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="1" column="4" alignment="Qt::AlignRight">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>My Grid</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>WPM</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="heardListComboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>95</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>List of stations used to complete macro substitutions. {DS}</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="macroToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMAC0"/>
   <addaction name="separator"/>
   <addaction name="actionMAC1"/>
   <addaction name="separator"/>
   <addaction name="actionMAC2"/>
   <addaction name="separator"/>
   <addaction name="actionMAC3"/>
   <addaction name="separator"/>
   <addaction name="actionMAC4"/>
   <addaction name="separator"/>
   <addaction name="actionMAC5"/>
   <addaction name="separator"/>
   <addaction name="actionMAC6"/>
   <addaction name="separator"/>
   <addaction name="actionMAC7"/>
   <addaction name="separator"/>
   <addaction name="actionMAC8"/>
   <addaction name="separator"/>
   <addaction name="actionMAC9"/>
   <addaction name="separator"/>
   <addaction name="actionMAC10"/>
   <addaction name="separator"/>
   <addaction name="actionMAC11"/>
   <addaction name="actionConfig"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLOG"/>
   <addaction name="separator"/>
   <addaction name="actionKILL_TX"/>
   <addaction name="separator"/>
   <addaction name="actionTUNE"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionMAC0">
   <property name="text">
    <string>MAC0</string>
   </property>
  </action>
  <action name="actionMAC1">
   <property name="text">
    <string>MAC1</string>
   </property>
  </action>
  <action name="actionMAC2">
   <property name="text">
    <string>MAC2</string>
   </property>
  </action>
  <action name="actionMAC3">
   <property name="text">
    <string>MAC3</string>
   </property>
  </action>
  <action name="actionMAC4">
   <property name="text">
    <string>MAC4</string>
   </property>
  </action>
  <action name="actionMAC5">
   <property name="text">
    <string>MAC5</string>
   </property>
  </action>
  <action name="actionMAC6">
   <property name="text">
    <string>MAC6</string>
   </property>
  </action>
  <action name="actionMAC7">
   <property name="text">
    <string>MAC7</string>
   </property>
  </action>
  <action name="actionMAC8">
   <property name="text">
    <string>MAC8</string>
   </property>
  </action>
  <action name="actionMAC9">
   <property name="text">
    <string>MAC9</string>
   </property>
  </action>
  <action name="actionMAC10">
   <property name="text">
    <string>MAC10</string>
   </property>
  </action>
  <action name="actionMAC11">
   <property name="text">
    <string>MAC11</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="icon">
    <iconset resource="qresource.qrc">
     <normaloff>:/images/images/gear-outline.png</normaloff>:/images/images/gear-outline.png</iconset>
   </property>
   <property name="text">
    <string>Config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionLOG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LOG</string>
   </property>
  </action>
  <action name="actionKILL_TX">
   <property name="text">
    <string>KILL TX</string>
   </property>
  </action>
  <action name="actionTUNE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TUNE</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qresource.qrc"/>
 </resources>
 <connections/>
</ui>
